src/main/java/com/altiscale/Util/ExecLoop.java

```diff
@@ -100,7 +100,7 @@ public void run() {
 
     while (shouldRestart()) {
       try {
-        execProcess = Runtime.getRuntime().exec(command);
+        execProcess = Runtime.getRuntime().exec(new String[] {"bash", "-c", command});
```

Comment:
we are now getting into unix-only world. Is it possible to use different exec() to make sure we can run this without bash installed?